jupyter-book/trajectories/rna_velocity.ipynb

```diff
@@ -15,53 +15,117 @@
    ]
```

Comment:
<p>It might be nice to add a simple figure below this that sketches the spliced/unspliced ratio etc. Maybe you have an idea or an example already? If so, Anna will happily create it and color it according to our current style. You don't have to create the figure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Do you mean phase portraits (Figure 4b <a href="Figure 4b [here](https://www.biorxiv.org/content/10.1101/2021.12.22.473434v1.full.pdf" target="_blank">here</a>), or intron/exons in the sequence (like <a href="like [here](https://www.google.com/search?q=introns+exons&amp;tbm=isch&amp;ved=2ahUKEwjtyNvf0bX6AhUknP0HHeKgD2EQ2-cCegQIABAA&amp;oq=introns+exons&amp;gs_lcp=CgNpbWcQAzIFCAAQgAQyBggAEB4QBTIGCAAQHhAIMgYIABAeEAgyBggAEB4QCDIGCAAQHhAIMgYIABAeEAgyBggAEB4QCDIGCAAQHhAIMgYIABAeEAhQ9QpY9QpgugxoAHAAeACAAdoBiAGYApIBBTEuMC4xmAEAoAEBqgELZ3dzLXdpei1pbWfAAQE&amp;sclient=img&amp;ei=z0UzY62lFKS49u8P4sG-iAY&amp;bih=752&amp;biw=1200&amp;rlz=1C5CHFA_enDE955DE955#imgrc=WZzd_uB08LaVoM" target="_blank">here</a> or <a href="https://www.google.com/search?q=introns+exons&amp;tbm=isch&amp;ved=2ahUKEwjtyNvf0bX6AhUknP0HHeKgD2EQ2-cCegQIABAA&amp;oq=introns+exons&amp;gs_lcp=CgNpbWcQAzIFCAAQgAQyBggAEB4QBTIGCAAQHhAIMgYIABAeEAgyBggAEB4QCDIGCAAQHhAIMgYIABAeEAgyBggAEB4QCDIGCAAQHhAIMgYIABAeEAhQ9QpY9QpgugxoAHAAeACAAdoBiAGYApIBBTEuMC4xmAEAoAEBqgELZ3dzLXdpei1pbWfAAQE&amp;sclient=img&amp;ei=z0UzY62lFKS49u8P4sG-iAY&amp;bih=752&amp;biw=1200&amp;rlz=1C5CHFA_enDE955DE955#imgrc=8i1YOjiYtm4l4M" target="_blank">here</a>). I personally thought of the "traditional" schematic as in Figure 2 <a href="Figure 4b here" target="_blank">here</a>.</p>

Reply:
I wasn't thinking of phase portraits (think we can discuss those later in the use-case) but rather introns/exons and how they relate to the spliced/unspliced ratio. Your second link is sadly broken.

Reply:
I assume that you mean the 2. figure of your paper though and think that it's a good one. Easy to recolor + slightly redraw by Anna.